packages/node_modules/@ciscospark/media-engine-webrtc/src/engine.js

```diff
@@ -721,6 +775,35 @@ export default class WebRTCMediaEngine {
     this[`sending${capitalize[kind]}`] = false;
   }
 
+  /**
+   * Compares target directions with senders and receivers and updates PC to match
+   * @returns {undefined}
+   */
+  updateLocalMediaToTargetDirection() {
+    ['audio', 'video'].forEach((kind) => {
+      const targetDirection = getTargetMediaDirection(this, kind);
```

Comment:
This may be improved with some comments. The variables are descriptive, but I had to read a couple of times because of the branching.